internal/catalogmetadata/client/client_test.go

```diff
@@ -28,7 +28,9 @@ func defaultCatalog() *catalogd.ClusterCatalog {
 			ResolvedSource: &catalogd.ResolvedCatalogSource{Image: &catalogd.ResolvedImageSource{
 				Ref: "fake/catalog@sha256:fakesha",
 			}},
-			ContentURL: "https://fake-url.svc.local/all.json",
+			URLs: &catalogd.ClusterCatalogURLs{
+				Base: "https://fake-url.svc.local/catalogs/catalog-1/api/v1/all.json",
```

Comment:
```suggestion
				Base: "https://fake-url.svc.local/catalogs/catalog-1/api/v1/all",
```

Reply:
Actually, this one should be 
```suggestion
				Base: "https://fake-url.svc.local/catalogs/catalog-1",
```
but good catch!
It's clearly not actually salient to the test suite, since no one is using it (or they would fail with the updates).